packages/widgetbook_generator/build.yaml

```diff
@@ -1,56 +1,14 @@
 builders:
-  theme_builder:
-    import: "package:widgetbook_generator/builder.dart"
-    builder_factories: ["themeBuilder"]
-    build_extensions: { ".dart": [".theme.widgetbook.json"] }
-    auto_apply: dependents
-    runs_before: [":widgetbook_builder"]
-    build_to: cache
-  locales_builder:
-    import: "package:widgetbook_generator/builder.dart"
-    builder_factories: ["localesBuilder"]
-    build_extensions: { ".dart": [".locales.widgetbook.json"] }
-    auto_apply: dependents
-    runs_before: [":widgetbook_builder"]
-    build_to: cache
-  localization_delegates_builder:
-    import: "package:widgetbook_generator/builder.dart"
-    builder_factories: ["localizationDelegatesBuilder"]
-    build_extensions: { ".dart": [".delegates.widgetbook.json"] }
-    auto_apply: dependents
-    runs_before: [":widgetbook_builder"]
-    build_to: cache
   story_builder:
```

Comment:
Can you see how we can rename this from `story_builder` to `use_case_builder` so that the naming is consistent across the application.